src/parser/jobs/nin/Cooldowns.js

```diff
@@ -0,0 +1,39 @@
+import CoreCooldowns from 'parser/core/modules/Cooldowns'
+import ACTIONS from 'data/ACTIONS'
+
+export default class Cooldowns extends CoreCooldowns {
+	static cooldownOrder = [
+		{
+			name: 'Ninjutsu',
+			actions: [
+				{
+					name: 'Mudra',
```

Comment:
Kinda nit-picky here but we should pluralize this and call it `Mudras`, I think. Also, we should probably add i18n into this.